### PR TITLE
WIP: M1 printer master

### PR DESCRIPTION
**Pull request by filipkotoucek**

### CODE REVIEW

File: src/assets/m1/prusaslicer.svg [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (Lowentwickler) [inline src/assets/m1/prusaslicer.svg:1]: This one is not optimized

File: src/assets/m1/refill_color.svg [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Lowentwickler) [inline src/assets/m1/refill_color.svg:1]: not optimized svg

File: src/assets/m1/remote_url.svg [hunk 3]
```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (Lowentwickler) [inline src/assets/m1/remote_url.svg:1]: not optimized svg

File: src/assets/m1/speed_color.svg [hunk 4]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Lowentwickler) [inline src/assets/m1/speed_color.svg:1]: not optimized svg

File: src/auth.js [hunk 5]
```diff
@@ -45,14 +45,14 @@ export const askApiKey = () =>
  * Create the readers with apiKey if exist and Accept keys
  * @param {string?} accept Accept header, default is `application/json`
  */
-const getHeaders = (accept) => {
+const getHeaders = (accept = "application/json") => {
```
Comment (Lowentwickler) [inline src/auth.js:48]: why did you remove this?

Comment (filipkotoucek) [reply src/auth.js:48]: The functionality is the same. Default value is more transparent I think than https://github.com/prusa3d/Prusa-Link-Web/blob/master/src/auth.js#L52

Comment (Lowentwickler) [reply src/auth.js:48]: Got it. I misread it.

File: spec/openapi.yaml [hunk 6]
```diff
@@ -732,6 +725,25 @@ paths:
...
         403:
           $ref: "#/components/responses/Unauthorized"
+    delete:
+      summary: Terminate the current file download process.
+      responses:
+        204:
+          description: No Content
+                       No Error
+
+  /api/download/{target}:
```
Comment (Lowentwickler) [inline spec/openapi.yaml:735]: [question]
We also have `/api/downloads/{target}`. Aren't we doing something wrong?

**Review by Lowentwickler — APPROVED** [review]
Here are few things regarding removal of the default values that bothers me. I will test them on real HW and fix them on BE if necessary.

File: src/assets/m1/prusaslicer.svg (hunk 1 above)
Comment (filipkotoucek) [reply src/assets/m1/prusaslicer.svg]: Should be fixed by 3b9ea159

File: src/assets/m1/refill_color.svg (hunk 2 above)
Comment (filipkotoucek) [reply src/assets/m1/refill_color.svg]: Should be fixed by 3b9ea15

File: src/assets/m1/remote_url.svg (hunk 3 above)
Comment (filipkotoucek) [reply src/assets/m1/remote_url.svg]: Should be fixed by 3b9ea15

File: src/assets/m1/speed_color.svg (hunk 4 above)
Comment (filipkotoucek) [reply src/assets/m1/speed_color.svg]: Should be fixed by 3b9ea15

File: spec/openapi.yaml (hunk 6 above)
Comment (filipkotoucek) [reply spec/openapi.yaml:735]: `/api/downloads` this endpoint is octoprint thing. They even do not mention it as separate endpoint. https://docs.octoprint.org/en/master/api/files.html?highlight=downloads. It is referenced in openapi.yaml only for completeness. AnywayIt is used for downloading files from printer to browser. Simply when you click on download button in project list.

`/api/download` as I understand is used for remote URL download (URL -> printer)

Comment (filipkotoucek) [reply spec/openapi.yaml:735]: I'm no expert on format of openapi.yaml. But GET /api/download is used without any other parameter so it seemed more logical to separate GET https://github.com/prusa3d/Prusa-Link-Web/blob/8609d746b92348a685d7bb26ee61db3c91422e8c/spec/openapi.yaml#L688 and POST https://github.com/prusa3d/Prusa-Link-Web/blob/8609d746b92348a685d7bb26ee61db3c91422e8c/spec/openapi.yaml#L735 to two endpoints
